src/neo-vm/ExecutionEngine.cs

```diff
@@ -70,6 +71,21 @@ public ExecutionEngine()
 
         protected virtual void ContextUnloaded(ExecutionContext context)
         {
+            if (InvocationStack.Count == 0)
+            {
+                CurrentContext = null;
+                EntryContext = null;
+            }
+            else
+            {
+                CurrentContext = InvocationStack.Peek();
+            }
+            if (context.StaticFields != null && context.StaticFields != CurrentContext?.StaticFields)
+            {
+                context.StaticFields.ClearReferences();
```

Comment:
If we have :

``` 
ContextA
ContextB
ContextA (as clone)
``` 

We will clean the StaticFields of `ContextA`, and there are another one in the stack, i will think if it's possible this behaviour.

Reply:
How is this done?

```cs
0: ContextA 
1: ContextB   
2: ContextA (as clone)  <---- Coud we clone the second top context?
```

Reply:
No. It can't be.